vortex-api/src/main/java/mil/army/usace/hec/vortex/VortexGrid.java

```diff
@@ -77,6 +77,32 @@ public static class VortexGridBuilder {
         private ZonedDateTime endTime;
         private Duration interval;
         private VortexDataType dataType;
+        private VortexVariable variable;
+
+        public VortexGridBuilder() {
+            // Empty constructor
+        }
+
+        public VortexGridBuilder(VortexGrid copy) {
```

Comment:
Would this copy constructor be useful? It does shallow copy of a VortexGrid instance, and should be accompanied by `toBuilder()`, which returns a `VortexGridBuilder` instance with the fields copied. Example of usage: `RasterUtils`